lis3dh.test.nut

```diff
@@ -0,0 +1,524 @@
+// MIT License
```

Comment:
Please don't include the library text file into the tests. Use *Builder* `@include` statements instead.